### PR TITLE
Documentation Horus API

### PR DESCRIPTION
**Pull request by RamonHossein**
Structure of Documentation
- Introduction
- Version
- Installation
- Usage
  (Login,
  Profile,
  Resource,
  telephone,
  Address)
- License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,129 +1,5 @@
...
-
-* **Slate is just Markdown** — when you write docs with Slate, you're just writing Markdown, which makes it simple to edit and understand. Everything is written in Markdown — even the code samples are just Markdown code blocks!
-
-* **Write code samples in multiple languages** — if your API has bindings in multiple programming languages, you easily put in tabs to switch between them. In your document, you'll distinguish different languages by specifying the language name at the top of each code block, just like with Github Flavored Markdown!
-
-* **Out-of-the-box syntax highlighting** for [almost 60 languages](http://rouge.jayferd.us/demo), no configuration required.
-
-* **Automatic, smoothly scrolling table of contents** on the far left of the page. As you scroll, it displays your current position in the document. It's fast, too. We're using Slate at TripIt to build documentation for our new API, where our table of contents has over 180 entries. We've made sure that the performance remains excellent, even for larger documents.
```
Comment (rebyoliveira) [inline README.md]: What's the mean of 'TripIt' ? 
"We're using Slate at TripIt to build documentation for our new API"

Comment (fernandes) [inline README.md:25]: TripIt is the guys who made "Slate" we can use:  "We are using Slate to document our API"

File: source/index.md [hunk 2]
```diff
@@ -16,153 +10,204 @@ includes:
...
+ Command Line: `ruby bin/horus-cli create addresses '"street" => "Rua", "number" =>  "55", "complement" => "Casa", "zipcode" => "37500344", "neighborhood" => "Bairro", "city" => "Cidade", "state" => "Estado", "country" => "Pais"' clients "id"`
 
-```ruby
-require 'kittn'
+* Update
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get(2)
-```
+ You also can update an address using a command line, with the id of the assress, or accessing a route, both cases are described below.
```
Comment (rebyoliveira) [inline source/index.md]: Correcting: 'with the id of the assress' - Should be "with the id of the address"

**Comment by rebyoliveira** [comment]
Finished review. Just fix the above point.

File: source/index.md [hunk 3]
```diff
@@ -16,153 +10,204 @@ includes:
...
+
+ Authentication as User allows the user to list and update information about yourself account.
+
+# Version
+
+ You will be able to discover the version of the Horus API that is used by the command line below.
+
+ `ruby bin/horus-cli version`
+
+ Currently the API is in version 0.1.0.
```
Comment (fernandes) [inline source/index.md]: v1

File: source/index.md [hunk 4]
```diff
@@ -16,153 +10,204 @@ includes:
...
+
+# Installation
+
+# Usage
+
+## Login
+
+ First of all, you should do the login using an email and a password already registered, along with the login you must specify which is your level of authenticating. The System Horus API currently has three types of authenticating how already explained above.
+
+ Rota: `POST "/oauth/token"`
```
Comment (fernandes) [inline source/index.md]: Route

File: source/index.md [hunk 5]
```diff
@@ -2,13 +2,7 @@
 title: API Reference
 
 language_tabs:
```
Comment (fernandes) [inline source/index.md:4]: actually you are doing `shell` not `ruby`, no?

**Comment by fernandes** [comment]
@RamonHossein instead of `ruby bin/horus-cli` you can use `horus-cli` because once gem is installed on system use can access the binary directly

File: source/index.md (hunk 5 above)
Comment (RamonHossein) [inline source/index.md:4]: Ok. I thought that in what language the API was made, but is in what language the user can access the API.

Comment (fernandes) [inline source/index.md:4]: I think its about in what languages you have examples

**Pull request merged**